### PR TITLE
Log deprecation message for recipes using easy_install.

### PR DESCRIPTION
**Pull request by remeh**

### CODE REVIEW

File: recipes/ddtrace-python.rb [hunk 1]
```diff
@@ -6,6 +6,11 @@
 # NB: This recipe is compatible with Chef <= 12 only. If you're using Chef 13 or higher
 # please refer to the README of the cookbook.
 
+log 'message' do
+  message 'This recipe will be deprecated in version 3.0.0 of the cookbook. See README.md chapter ddtrace-python for more infos.'
```
Comment (olivielpeau) [inline recipes/ddtrace-python.rb]: `chapter` -> `section`

`infos` -> `info`

Also, can you remind the name of the cookbook in the message? (`datadog cookbook`). Generally useful.

File: recipes/ddtrace-python.rb [hunk 2]
```diff
@@ -6,6 +6,11 @@
 # NB: This recipe is compatible with Chef <= 12 only. If you're using Chef 13 or higher
 # please refer to the README of the cookbook.
 
+log 'message' do
```
Comment (olivielpeau) [inline recipes/ddtrace-python.rb]: any reason to use the resource instead of `Chef::Log`? (genuine question, didn't know about the resource)

Comment (remeh) [reply recipes/ddtrace-python.rb]: Not any, I've changed to use Chef::Log as I see there is already many usage of it in the repo, I prefer to stick with known things.

File: recipes/ddtrace-python.rb (hunk 1 above)
Comment (remeh) [reply recipes/ddtrace-python.rb]: thanks 👍 
I've also repeated the recipe name.

**Pull request merged**